coupling/delwaq/generate.py

```diff
@@ -0,0 +1,359 @@
+"""Setup a Delwaq model from a Ribasim model and results."""
+
+import csv
+import shutil
+from datetime import timedelta
+from pathlib import Path
+
+import networkx as nx
+import numpy as np
+import pandas as pd
+import ribasim
+from jinja2 import Environment, FileSystemLoader
+from util import (
+    strfdelta,
+    ugrid,
+    write_flows,
+    write_pointer,
+    write_volumes,
+)
+
+delwaq_dir = Path(__file__).parent
+
+env = Environment(autoescape=True, loader=FileSystemLoader(delwaq_dir / "template"))
+
+# Add evaporation edges, so mass balance is correct
+# To simulate salt increase due to evaporation, set to False
+USE_EVAP = True
+
+
+def generate(modelfn: Path) -> tuple[nx.DiGraph, set[str]]:
```

Comment:
For consistency with Ribasim Python it would perhaps be better to use `toml_path` rather than `modelfn` everywhere.